content/influxdb/v2.0/reference/flux/stdlib/influxdb-v1/fieldsascols.md

```diff
@@ -0,0 +1,59 @@
+---
+title: v1.fieldsAsCols() function
+description: The v1.fieldsAsCols() function is pivots a table and automatically aligns fields within each input table that have the same timestamp.
+aliases:
+  - /influxdb/v2.0/reference/flux/functions/inputs/fromrows
+  - /influxdb/v2.0/reference/flux/functions/transformations/influxfieldsascols
+  - /influxdb/v2.0/reference/flux/functions/influxdb-v1/fieldsascols/
+  - /influxdb/v2.0/reference/flux/functions/influxdb-schema/fieldsascols/
+menu:
+  influxdb_2_0_ref:
+    name: v1.fieldsAsCols
+    parent: InfluxDB v1
+weight: 301
+introduced: 0.16.0
+deprecated: 0.88.0
+---
+
+{{% warn %}}
+`v1.fieldsAsCols()` was deprecated in **Flux v0.88.0** in favor of
+[`schema.fieldsAsCols()`](/influxdb/v2.0/reference/flux/stdlib/influxdb-schema/fieldsascols/).
+{{% /warn %}}
+
+The `v1.fieldsAsCols()` function is a special application of the `pivot()` function that
```

Comment:
link to pivot() function?

Reply:
how is it special? Could we get away w saying 
...function aligns fields in each...

Reply:
It's special because it applies to the specific use case of pivoting on the `_field` and `_time` columns.